lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -168,9 +173,9 @@ data JormungandrLayer m = JormungandrLayer
     , postMessage
         :: (Tx, [TxWitness])
         -> ExceptT ErrPostTx m ()
-    , getInitialFeePolicy
+    , getInitialConfigParams
```

Comment:
I'd recommend to avoid too generic terms where we can (like `ConfigParams`). There are many things that requires `ConfigParams`, SQLite does, Servant client does etc ... 
I think it's best if we use more specific names where we can:

```hs
getInitialBlockchainParameters
```

Reply:
done